test/utils.py

```diff
@@ -5,6 +5,12 @@
 dtypes = [torch.half, torch.float, torch.double, torch.int, torch.long]
 grad_dtypes = [torch.half, torch.float, torch.double]
 
+import torch_scatter
+if torch_scatter.__version__ > '2.0.9':
```

Comment:
I don't think you can use `>` like this in strings. You would test against the individual integers.

Reply:
This is similar with https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/profile/profiler.py#L107.

Reply:
I think then this is bugged as well. You can see that it breaks here:
```
'2.0.10' > '2.0.9'
```

Reply:
Use `version.parse(torch_scatter.__version__) > version.parse("2.0.9")` can help. This is from PyTorch.

Reply:
Cool, thanks!